app/src/main/java/com/android/unio/ui/association/AssociationProfile.kt

```diff
@@ -496,7 +497,15 @@ private fun AssociationEvents(
   }
   if (isAdmin) {
     Button(
-        onClick = { navigationAction.navigateTo(Screen.EVENT_CREATION) },
+        onClick = {
```

Comment:
Nice modification to ensure that the user is connected for before adding events !